### DISCUSSION TITLE
Titan Network, a high-performance DePIN public blockchain boasting tens of millions of TPS, captivates the Cosmos ecosystem with fervor!

### DISCUSSION THREAD

**Discussion by MaryTitan** [Titannet-dao/Community#5] [RetroPGF]
👉_Titan Network welcomes any initiative or contribution that promotes its presence and extends its influence within the Cosmos ecosystem, rewarding such efforts with [RetroPGF](https://github.com/Titannet-dao/Community/discussions/3) awards!_ 👈

🔥We're thrilled to embark on a groundbreaking journey with the Cosmos Hub—creating our very own blockchain. Presently in the internal testing phase, we aim to roll out a testnet (the third round) related to our blockchain by Q2 2024. Our mission is to tap into the vast potential of the Cosmos ecosystem to fundamentally transform the delivery and experience of decentralized services.

**✨Why Cosmos?**
Dubbed the "Internet of Blockchains," Cosmos stands unrivaled in its interoperability, scalability, and flourishing ecosystem. These benefits make it the ideal foundation for Titan Network's blockchain, built on the Cosmos SDK. This not only ensures enhanced flexibility but also aids in managing the rising gas costs associated with increasing user engagement. Presently, Titan Network is actively engaged in the first round of the testnet, Huygens Testnet, featuring PC clients and command-line testing. This initiative has attracted 60,000 nodes from 45 countries worldwide, all connected simultaneously within just a week of its launch. As we prepare to support mobile clients in the second round of the testnet, we anticipate a significant reduction in entry barriers, paving the way for the participation of millions of nodes.

In our initial stages, we plan to employ Cosmos's native POS or DPOS consensus mechanisms, eventually transitioning to the narwhal + bullshark consensus algorithm before our mainnet launch. This algorithm, also utilized by SUI, is capable of supporting tens of thousands of TPS in real-world scenarios, with lab tests indicating potential peaks of 300,000 to 400,000 TPS—ideal for a high-performance blockchain. To accommodate the need for substantial resource support, Titan Network also envisions the future implementation of flexible subnets, allowing for the processing of up to tens of millions of TPS while maintaining minimal Gas fees. This positions Titan as a foundational web3 platform capable of supporting large-scale web2 applications and even the metaverse.

**💎Your Wisdom and Passion Are Needed**
Realizing this ambitious vision requires more than just technology—it demands a community of innovators, builders, and believers. We extend a warm invitation to developers, blockchain enthusiasts, and seasoned members of the Cosmos community to join us on this historic venture.

**💫How Can You Get Involved?**

**💓Feedback and Ideas:** Contribute your perspectives, advice, and critiques of our strategies. Engage in discussions on leveraging the Cosmos ecosystem to create truly exceptional solutions.

**💓Development and Testing:** Join our testnet to help identify and resolve issues, suggest improvements, and push the limits of our blockchain within the Cosmos ecosystem.

**💓Planning and Promotion:** Get involved in proposal planning, discussions, and marketing efforts, especially those aimed at enhancing Titan Network's visibility and impact within the Cosmos ecosystem.

**💓Spreading the Word:** Amplify our reach by sharing our narrative, updates, and accomplishments, extending beyond just the Cosmos community.

**💪Why Us?**
Contributions will be recognized with RetroPGF awards from Titan Network! Joining us means more than being part of another blockchain initiative; you'll be integral to a revolutionary project that's redefining access to and the experience of decentralized services, all within the Cosmos ecosystem.

**🔥Let's Create History Together**
Share your ideas, express your interest in collaboration, or learn more about our project. Your participation is invaluable to us. Let's collaborate, innovate, and achieve success together.

**Comment by thompsonng** [comment]
This is great. I am willing to join in the testnet and spread the word to more people. I am currently running your testnet product. Quite cool though I don't understand much because I don't have background in tech

**Reply by MaryTitan** [reply]
Thank you for your support! It is quite easy to run a Titan node as long as you follow the tutorials on our Gitbook. We are always happy to hear comments and suggestions from the community! :)

**Reply by 14mi9** [reply]
nice

**Comment by lucas-ron** [comment]
I want to participate in the test, but I can't register an account at present, what should I do

**Reply by MaryTitan** [reply]
We were encountering some issues earlier and you should be able to register for an account now. Please let me know if there are still problems. Thank you.

**Comment by dmytroraise** [comment]
Could be intresting. THanks for info!

**Reply by MaryTitan** [reply]
Thanks for your interest! Please participate in the testnet and contribute. We provide good rewards with all these! :)

**Comment by quinnwencn** [comment]
Hi, I am interested in joining this wonderful project! But I failed to bind my id using docker in CentOS7.  Could you help me solve this problem? 
Here is the error message:
```
024-05-13T02:54:28.997Z	ERROR	main	titan-edge/main.go:108	Post "https://api-test1.container1.titannet.io/api/v2/device/binding": dial tcp: lookup api-test1.container1.titannet.io: i/o timeout

```

**Reply by MaryTitan** [reply]
Please try to restart the program, run it for a while then bind the code.

**Reply by quinnwencn** [reply]
Solved by restarting my machine, thanks!

**Comment by Myking070323** [comment]
I just can support u with share you projects dev .. 
We hope u projects success ...

**Reply by MaryTitan** [reply]
Fantastic! Have you joined our volunteer program? Please fill out this form (https://tally.so/r/nr696X) if you are interested in applying for the volunteer program!

**Comment by Myking070323** [comment]
You are welcome Dev .. i hope u projects succes .. and go to the moon! :D

Pada Rab, 29 Mei 2024 13.16, Robert ***@***.***> menulis:

**Comment by haidarabdillah** [comment]
anyway to create transaction or do on chain using package js? maybe sdk ? i want to try it
